src/components/ButtonPanel.js

```diff
@@ -1,71 +1,35 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useState } from 'react';
 import Button from './Button';
 
-
-class ButtonPanel extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  handleClick(buttonName) {
-    const { clickHandler } = this.props;
-    clickHandler(buttonName);
-  }
-
-  renderButton(name) {
-    return (
-      <Button
-        buttonName={name}
-        clickHandler={this.handleClick}
-      />
-    );
-  }
-
-  render() {
-    return (
-      <div className="btn-panel">
-        <div className="group1 board-row">
-          {this.renderButton('AC')}
-          {this.renderButton('+/-')}
-          {this.renderButton('%')}
-          {this.renderButton('÷', true)}
-        </div>
-        <div className="group2 board-row">
-          {this.renderButton('7')}
-          {this.renderButton('8')}
-          {this.renderButton('9')}
-          {this.renderButton('x', true)}
-        </div>
-        <div className="group3 board-row">
-          {this.renderButton('4')}
-          {this.renderButton('5')}
-          {this.renderButton('6')}
-          {this.renderButton('-', true)}
+const ButtonPanel = () => {
+  const [buttons] = useState([
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ]);
+
+  const [groupIndices] = useState([1, 2, 3, 4]);
+
+  const [operationCharacters] = useState(['÷', 'X', '-', '+', '=']);
+
+  return (
+    <div className="btn-panel">
+      {buttons.map((group, outerIndex) => (
+        <div className="btn-group" key={groupIndices[outerIndex]}>
+          {group.map(character => {
+            if (operationCharacters.includes(character)) {
+              return <Button key={character} name={character} />;
+            } if (character === '0') {
+              return <Button key={character} name={character} color wide />;
+            }
+            return <Button key={character} name={character} color />;
```

Comment:
Now that `color` is turned into STRING in <Button /> component, you supposed to pass a string here at the PANEL (not a boolean)

Reply:
WHen you solve the above observations, it will eliminate some of the console errors of your app below. Try to solve these console errors
![Screen Shot 2020-04-25 at 8 48 09 PM](https://user-images.githubusercontent.com/40712459/80289284-96214280-8735-11ea-878d-d5d261d4085f.png)